### PR TITLE
installer scripts

### PR DESCRIPTION
**Pull request by NaveenSidhartha**
all remaining installer scripts

### CODE REVIEW

File: installer_scripts/cop_install_podnet_a.py [hunk 1]
```diff
@@ -1 +1,348 @@
...
+    firewall_rules = [
+        # 3.1.1 Inbound IPv4
+
+        # "lo" all accept
+        {'order': 3111, 'version': '4', 'iiface': 'lo', 'oiface': '', 'protocol': 'any', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['127.0.0.0/24'], 'port': []},
+        # Ping Accept on Public interface
+        {'order': 3112, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'icmp', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['any'], 'port': []},
+        # DNS Accept on Public interface
+        {'order': 3113, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'dns', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['any'], 'port': []},
+        # VPN Accept on Public interface CPE since it has Region in blend
```
Comment (maria-walsh) [inline installer_scripts/cop_install_podnet_a.py:211]: As this is a COP Podnet A installation script, why is the VPN firewall rule required?

File: installer_scripts/cop_install_podnet_b.py [hunk 2]
```diff
@@ -1 +1,347 @@
...
+        # 3.1.1 Inbound IPv4
+
+        # "lo" all accept
+        {'order': 3111, 'version': '4', 'iiface': 'lo', 'oiface': '', 'protocol': 'any', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['127.0.0.0/24'], 'port': []},
+        # Ping Accept on Public interface
+        {'order': 3112, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'icmp', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['any'], 'port': []},
+        # DNS Accept on Public interface
+        {'order': 3113, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'dns', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['any'], 'port': []},
+        # VPN Accept on Public interface CPE since it has Region in blend
+        {'order': 3114, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'vpn', 'action': 'accept', 'log': True, 'source': ['any'], 'destination': [ipv4_link_cpe], 'port': []},
```
Comment (maria-walsh) [inline installer_scripts/cop_install_podnet_b.py:211]: Is this required for COP?

File: installer_scripts/cop_reinstall_podnet_a.py [hunk 3]
```diff
@@ -1,2 +1,347 @@
...
+        # 3.1.1 Inbound IPv4
+
+        # "lo" all accept
+        {'order': 3111, 'version': '4', 'iiface': 'lo', 'oiface': '', 'protocol': 'any', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['127.0.0.0/24'], 'port': []},
+        # Ping Accept on Public interface
+        {'order': 3112, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'icmp', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['any'], 'port': []},
+        # DNS Accept on Public interface
+        {'order': 3113, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'dns', 'action': 'accept', 'log': False, 'source': ['any'], 'destination': ['any'], 'port': []},
+        # VPN Accept on Public interface CPE since it has Region in blend
+        {'order': 3114, 'version': '4', 'iiface': 'public0', 'oiface': '', 'protocol': 'vpn', 'action': 'accept', 'log': True, 'source': ['any'], 'destination': [ipv4_link_cpe], 'port': []},
```
Comment (maria-walsh) [inline installer_scripts/cop_reinstall_podnet_a.py:211]: is it required?

**Review by maria-walsh — CHANGES_REQUESTED** [review]

File: installer_scripts/cop_install_podnet_a.py (hunk 1 above)
Comment (NaveenSidhartha) [reply installer_scripts/cop_install_podnet_a.py:211]: Currently its use is in Region only, not required for COP so removing it

**Pull request merged**